ONSdigital/eq-survey-runner
# Build individual census

### What is the context of this PR?
Initial build of the individual Census questionnaire using Jsonnet. 

### How to review
- Build the schema using `./scripts/build_schemas.sh`
- Use the flow documented in overflow or the existing [schema](https://github.com/ONSdigital/eq-census-schemas/blob/census-individual/census_individual.json) to verify the flow is correct 
- Check variants for proxy, region, over/under 16 and current/past jobs
- Check for correct placeholder resolution
